sql_generators/glean_usage/templates/baseline_clients_last_seen_v1.checks.sql

```diff
@@ -16,9 +16,6 @@
   #warn
   {{ is_unique(["client_id"], where="submission_date = @submission_date") }}
 
-  #warn
-  {{ min_row_count(1, where="submission_date = @submission_date") }}
-
```

Comment:
There are multiple apps that don't have any data some days

Reply:
I don't think this is the right approach here. Just because some apps don't expect data here it doesn't mean we shouldn't do the check for those apps where we do expect to have new entries. I have an alternative PR here where the baseline checks should just not be copied over for specific apps:

https://github.com/mozilla/bigquery-etl/pull/4670